synapse/storage/databases/main/room.py

```diff
@@ -1396,7 +1478,7 @@ async def maybe_store_room_on_outlier_membership(
             insertion_values={
                 "room_version": room_version.identifier,
                 "is_public": False,
-                "creator": "",
+                "creator": room_creator,
```

Comment:
Instance #2 where we insert into the `rooms` table and need to make sure `creator` is also populated